documentation/src/main/asciidoc/user_guide/eviction.adoc

```diff
@@ -1,16 +1,24 @@
+Infinispan supports eviction of entries, such that you do not run out of memory.
+Eviction is typically used in conjunction with a cache store, so that entries are
+not permanently lost when evicted, since eviction only removes entries from
+memory and not from cache stores or the rest of the cluster.
 
 TIP:  Passivation is also a popular option when using eviction, so that only a single copy of an entry is maintained - either in memory or in a cache store, but not both. The main benefit of using passivation over a regular cache store is that updates to entries which exist in memory are cheaper since the update doesn't need to be made to the cache store as well.
 
-IMPORTANT: that eviction occurs on a _local_ basis, and is not cluster-wide.  Each node runs an eviction thread to analyse the contents of its in-memory container and decide what to evict. Eviction does not take into account the amount of free memory in the JVM as threshold to  starts evicting entries. You have to set maxEntries attribute of the eviction element to be greater than zero in order for eviction to be turned on. If maxEntries is too large you can run out of memory. maxEntries attribute will probably take some tuning in each use case.
+IMPORTANT: that eviction occurs on a _local_ basis, and is not cluster-wide.  Each node runs an eviction thread to analyse the contents of its in-memory container and decide what to evict. Eviction does not take into account the amount of free memory in the JVM as threshold to  starts evicting entries. You have to set size attribute of the eviction element to be greater than zero in order for eviction to be turned on. If size is too large you can run out of memory. size attribute will probably take some tuning in each use case.
 
 === Enabling Eviction
-Eviction is configured by adding the link:http://docs.jboss.org/infinispan/{infinispanversion}/configdocs/infinispan-config-{infinispanversion}.html[`<eviction />`] element to your `<*-cache />` configuration sections or using link:{javadocroot}/org/infinispan/configuration/cache/EvictionConfigurationBuilder.html[EvictionConfigurationBuilder] API programmatic approach.
+Eviction is configured by adding the
+link:http://docs.jboss.org/infinispan/{infinispanversion}/configdocs/infinispan-config-{infinispanversion}.html[`<eviction />`]
+element to your `<*-cache />` configuration sections or using
+link:{javadocroot}/org/infinispan/configuration/cache/EvictionConfigurationBuilder.html[EvictionConfigurationBuilder]
+API programmatic approach.
 
 All cache entry are evicted by piggybacking on user threads that are hitting the cache.
-Periodic pruning of expired cache entries from cache is done on a dedicated thread which is turned on by enabling reaper in expiration configuration element/API.
+Periodic pruning of expired cache entries from cache is done on a dedicated thread
+which is turned on by enabling reaper in expiration configuration element/API.
```

Comment:
Yeah I wasn't trying to change the overall flow of the document at all.  But I can move these over.